_haikus/SasSy.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: SaaSy
+author: Julian
+---
+
+"No software" the goal  
```

Comment:
Make sure you have two spaces at the end of each line of the haiku, or else use the `<br>` HTML tag to break the line.